tests/test_SdssShape.py

```diff
@@ -129,6 +139,47 @@ def testMeasureBadPsf(self):
             self._checkShape(result, record)
             self.assertTrue(result.getFlag(lsst.meas.base.SdssShapeAlgorithm.PSF_SHAPE_BAD.number))
 
+    def testMonteCarlo(self):
+        """Test an ideal simulation, with no noise.
```

Comment:
"with no noise" doesn't seem right.  Maybe "with known noise" or "with controlled noise"?

Reply:
This was just copy-pasted from other MonteCarlo tests. But you're right that this is not accurate and will change.